pkgs/development/libraries/flatpak/default.nix

```
@@ -79,7 +79,7 @@ stdenv.mkDerivation rec {
     # Hardcode paths used by Flatpak itself.
     (substituteAll {
       src = ./fix-paths.patch;
-      p11 = p11-kit;
+      p11 = p11-kit.dev;
```

Could we instead use 

```suggestion
      p11kit = "${p11-kit.dev}/bin/p11-kit";
```

and substitute `@p11kit@` in the patch?

---

Did that.